src/game_models.cpp

```diff
@@ -73,3 +75,10 @@ QVariant GameListModel::data(const QModelIndex &index, int role) const
         return QVariant();
     }
 }
+
+void GameListModel::sortNames()
+{
+    std::sort(m_items.begin(), m_items.end(), [](Game *a, Game *b) {
+        return a->name() < b->name();
+    });
+}
```

Comment:
Have to read this https://doc.qt.io/qt-6/model-view-programming.html?

Reply:
Nope, but I just poked until I got something working lol

Reply:
Then better to remove this because GameListModel is more complicated than what it looks.

Reply:
Looking back at the URL, I don't quite see the concern. All it does is reorder the entries of games that way they go in alphabetical order. The game's folder path and all still matches the game.

So, if I had:

Game1
CUSA00001

Ame2
CUSA00002

It would sort as:

Ame2
CUSA00002

Game1
CUSA00001

You won't get an issue such as:

Ame2
CUSA00001 (Game1's folder)

Game1
CUSA00002 (Ame2's folder)

Reply:
How does the widget know it needs to update because the order of the items was changed?

Reply:
Hmm? You don't have to explicitly tell the widget anything. It reflects off of the model. If the model gets sorted, the widget's options get sorted as well.

Reply:
No it need some mechanism to tell the widget that the model has been updated. It you think it is happen automagically that mean you don't understand the how it is working yet.

Reply:
> No it need some mechanism to tell the widget that the model has been updated. It you think it is happen automagically that mean you don't understand the how it is working yet.

Despite it working already? Ugh, fine... I'll add resets...

Reply:
Because it is C++. Even if it work does not mean it is working correctly.